### PR TITLE
[LPZ-2023] add speakers and program

### PR DESCRIPTION
**Pull request by tebo**
*Please title your pull request in this format: The event name and year in the title of the PR, along with a description of what is being changed, i.e., `[CHI-2019] Add Bluth Company as a sponsor`*

If you are adding or removing organisers, please email info@devopsdays.org with the details of who is being added and removed, along with the full names and email addresses of the new team, so we can update Slack and the mailing list.

### CODE REVIEW

File: content/events/2022-la-paz/speakers/esteban-lima copy.md
Comment (adrianmoisey) [inline content/events/2022-la-paz/speakers/esteban-lima copy.md:1]: This file is in the 2022 directory, is that a mistake?

File: content/events/2022-la-paz/speakers/milton-hilara copy.md
Comment (adrianmoisey) [inline content/events/2022-la-paz/speakers/milton-hilara copy.md:1]: This file is in the 2022 directory, is that a mistake?

File: content/events/2022-la-paz/speakers/esteban-lima copy.md
Comment (tebo) [reply content/events/2022-la-paz/speakers/esteban-lima copy.md:1]: Yes, it's a mistake. I've already delete the file

File: content/events/2022-la-paz/speakers/milton-hilara copy.md
Comment (tebo) [reply content/events/2022-la-paz/speakers/milton-hilara copy.md:1]: Yes, it's a mistake. I've already delete the file

**Pull request merged**